packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2945,6 +2997,13 @@ class InputDecoration {
   ///  * [prefixIcon], which is the same but on the leading edge.
   final Widget? suffixIcon;
 
+  /// The alignment for [suffixIcon].
+  ///
+  /// Used to align the [suffixIcon] relative to the position of the text box.
+  ///
+  /// The default value is [VerticalAlignment.center].
+  final VerticalAlignment? suffixIconAlignment;
```

Comment:
Will this also affect `suffix` and not just `suffixIcon`?  Maybe we should rename all of this to just be `suffixAlignment` (same for prefix).

Reply:
It will affect `prefixIcon` and `suffixIcon` only.

Reply:
I think the aim of `prefix` and `suffix` is be right before and after the text.... Usually it's used in one line text fields too. So prefixAlignment to the center ? I don't think that might be the expected behaviour